superset/connectors/sqla/utils.py

```diff
@@ -209,9 +212,15 @@ def find_cached_objects_in_session(
     if not ids and not uuids:
         return iter([])
     uuids = uuids or []
+    try:
+        items = set(session)
+    except ObjectDeletedError:
+        logger.warning("ObjectDeletedError", exc_info=True)
+        return iter(())
```

Comment:
this was erroring for me when running integration tests, so I just thought it would be safe to catch and log the error in case it happens again.